graphql/schema/types/project.graphql

```diff
@@ -182,7 +182,6 @@ type Project {
   patchTriggerAliases: [PatchTriggerAlias!]
   perfEnabled: Boolean
   periodicBuilds: [PeriodicBuild!]
-  private: Boolean
```

Comment:
This being used on the frontend, you would first need to remove its use on the UI before deleting it here.

Reply:
Not deleting anything from the schema anymore 😅

Reply:
sgtm if you do decide to make some changes to gql feel free to tag me again.